pyproject.toml

```diff
@@ -0,0 +1,174 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "pghoard"
+authors = [
+  { name="Aiven", email="opensource@aiven.io" },
+  { name="Hannu Valtonen", email="hannu.valtonen@ohmu.fi" }
+]
+description = "PostgreSQL automatic backup/restore service daemon."
+readme = "README.rst"
+requires-python = ">=3.8"
+classifiers=[
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Information Technology",
+    "Intended Audience :: System Administrators",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Topic :: Database :: Database Engines/Servers",
+    "Topic :: Software Development :: Libraries",
+]
+license = { text = "Apache License 2.0" }
+dynamic = ["version"]
+dependencies = [
+    "cryptography",
+    "psycopg2-binary >= 2.8.0",
+    "pydantic",
+    "python-dateutil",
+    "python-snappy >= 0.5",
+    "python-systemd",
+    "requests >= 1.2.0",
+    "rohmu >= 1.0.7",
+    "zstandard >= 0.11.1",
+]
+
+[project.optional-dependencies]
+dev = [
+    "boto3",
+    "mock",
+    "mypy",
+    "pylint>=2.4.3,<=2.7.2",
+    "pylint-quotes",
+    "pytest",
+    "pytest-cov",
+    "pytest-mock",
+    "pytest-timeout",
+    "pytest-xdist",
+    "yapf==0.30.0",
+    "isort==5.7.0",
+    "coverage",
+    "coveralls",
+    "freezegun>=1.2",
+    "responses",
+    "unify",
+    "types-botocore",
+    "types-httplib2",
+    "types-mock",
+    "types-paramiko",
+    "types-psycopg2",
+    "types-python-dateutil",
+    "types-requests",
+    "types-six",
+]
+contraints = [
```

Comment:
I am not sure if this is used for anything, but you have a spelling error here, `contraints` -> `constraints`

Reply:
oh yeah this is used in the github workflow. i will fix